benches/benches/block_target_gas_set/contract.rs

```diff
@@ -0,0 +1,20 @@
+use crate::*;
```

Comment:
Do we need this file?

Reply:
I'll probably just keep it since we're gonna add it soon anyway.